micro-ui/web/micro-ui-internals/packages/modules/hrms/src/pages/createEmployee.js

```
@@ -146,12 +147,12 @@ const CreateEmployee = () => {
 
   const onSubmit = async (data) => {
     const hasCurrentAssignment = data?.Assignments?.some(assignment => assignment?.isCurrentAssignment === true); 
-    let selectedCity= data?.Jurisdictions?.boundary;
+    let selectedCity= data?.Jurisdictions?.[0]?.boundary;
```

Biome

[error] This let declares a variable that is only assigned once.

'selectedCity' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)